chapter-04/recipe-01/cxx-example/main.cpp

```diff
@@ -1,15 +1,16 @@
-#include "sum_integers.hpp"
-
 #include <iostream>
 #include <vector>
+#include <string>
+
+#include "sum_integers.hpp"
```

Comment:
Whooops. This slipped through. Might have to go through some of the other `main.cpp`-s at a later stage.

Reply:
I will write a script to detect these: https://github.com/bast/cmake-recipes/issues/313 :-)